moskito-extensions/moskito-notification-providers/src/main/java/net/anotheria/moskito/extensions/notificationproviders/BulkSMSNotificationProvider.java

```diff
@@ -127,16 +128,16 @@ private String formatQuery(final ThresholdAlert alert) {
     /**
      * Create SMS message.
      *
-     * @param alert {@link net.anotheria.moskito.core.threshold.alerts.ThresholdAlert}
-     * @param template {@link net.anotheria.moskito.extensions.notificationproviders.BulkSMSNotificationProvider#smsTemplate}
+     * @param alert {@link ThresholdAlert}
+     * @param template {@link BulkSMSNotificationProvider#smsTemplate}
      * @return SMS
      */
     private static String createMessage(final ThresholdAlert alert, final String template) {
         // TODO: message should be encoded to apropriate encoding
         //return alert.getThreshold().getName() + ": " + alert.getOldStatus() + "->" + alert.getNewStatus();
         String message;
         if(!StringUtils.isEmpty(template)) {
-            ThresholdAlertTemplate thresholdAlertTemplate = new ThresholdAlertTemplate(alert);
+            MailTemplate thresholdAlertTemplate = new ThresholdAlertTemplate(alert);
```

Comment:
As I commented earlier: ThresholdAlertTemplate is a concrete implementation of MailTemplate. If you decide to change the implementation, you may replace it without hassle. The abstract type reflects this possibility. I found a good explanation here: http://stackoverflow.com/a/9853116